e2e/next/src/next.test.ts

```diff
@@ -421,7 +435,7 @@ describe('Next.js Applications', () => {
       checkUnitTest: false,
       checkLint: false,
       checkE2E: false,
-      checkExport: true,
+      checkExport: false,
```

Comment:
We really don't care about exports at this point since it's a deprecated Next.js command.